subworkflows/local/utils_nfcore_mcmicro_pipeline/main.nf

```diff
@@ -148,6 +150,121 @@ workflow PIPELINE_COMPLETION {
     }
 }
 
+workflow UPDATE_FROM_OME {
```

Comment:
I think this should be moved to a separate subworkflow as this one is templated by nf-core tools. The less extra code we put in it the lower the chance of confusing the template merge bot and triggering merge conflicts. It should be as simple as pulling this `workflow` and the 2 new `include` lines above into a new file named like `subworkflows/local/update_from_ome/main.nf` or similar.